README.md

```diff
@@ -1,9 +1,71 @@
 [![Build Status](https://travis-ci.org/kubernetes-csi/driver-registrar.svg?branch=master)](https://travis-ci.org/kubernetes-csi/driver-registrar)
 # Cluster Driver Registrar
 
-A sidecar container that
+The cluster-driver-registrar is a sidecar container that creates a cluster-level
+[CSIDriver object](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/csi-api/pkg/crd/manifests/csidriver.yaml)
+for the CSI driver.
 
-1. Registers the containerized CSI driver with Kubernetes and create a CSIDriver object
+This sidecar container is only needed if you need one of the following Kubernetes
+features:
+
+<!-- TODO: Reference skip attach docs here -->
+* Skip attach: For drivers that don't support ControllerPublishVolume, this
+  eliminates the need to deploy the external-attacher sidecar
+<!-- TODO: Reference pod info docs here -->
+* Pod info on mount: This passes Kubernetes metadata such as Pod name and
+  namespace to the NodePublish call
+
+If you are not using one of these features, this sidecar container (and the
+creation of the CSIDriver Object is not required. However, it is still
```

Comment:
`creation of the CSIDriver Object)` <- Close parenthesis

Reply:
...I don't understand the comment. Why would there be a close parenthesis if there's no link? I figured since we're already linking the CSIDriver Object near the beginning it wouldn't be needed for every instance.